src/components/helpers/pretty-text-input.js

```diff
@@ -417,7 +417,12 @@ export default createReactClass({
 
     this.debouncedOnChangeAndTagCodeMirror = _.debounce(() => {
       this.isDebouncingCodeMirrorChange = false;
-      this.onChangeAndTagCodeMirror();
+
+      // If this.codeMirror no longer exists don't tag, to avoid this error:
+      // https://cdn.zapier.com/storage/photos/69d375365045c5fd6e9c7e2a68c43905.png
+      if (this.codeMirror) {
+        this.onChangeAndTagCodeMirror();
```

Comment:
By "tag", do we mean create a pretty pill?

Reply:
Yes. We create a [`PrettyTag`](https://github.com/zapier/formatic/blob/fix-bug-tagging-non-existent-code-mirror/src/components/helpers/pretty-tag.js#L19) element [here](https://github.com/zapier/formatic/blob/fix-bug-tagging-non-existent-code-mirror/src/components/helpers/pretty-text-input.js#L440-L444), and then tell CodeMirror to use that whenever it sees a certain `{{123__value}}` text value.